src/structure/Screen/Screen.jsx

```diff
@@ -0,0 +1,41 @@
+import { Component } from 'preact';
+
+import '../../Theme/Theme';
+import CSS from './styles.css';
+
+const Screen = (props) => {
+	const [lastClick, setLastClick] = useState({x:0, y:0})
+	const viewport = { width: 0, height: 0};
+
+	useEffect(() => {
+		viewport.width = window.innerWidth;
+		viewport.height = window.innerHeight;
+		window.addEventListener('resize', handleOnWindowResize);
+		return () => {
```

Comment:
(simple return) please refactor this so that you assign this expression to a new `const result` and then separately `return result;`